content/swagger/resources/measurement_families/definitions/measurement_family.yaml

```diff
@@ -0,0 +1,81 @@
+type: object
+required: ["code"]
+properties:
+  code:
+    type: string
+    description: Measurement family code
+    x-immutable: true
+  labels:
+    type: object
+    description: Measurement family labels for each locale
+    x-validation-rules: The `localeCode` is the code of an existing and activated locale
+    default: {}
+    properties:
+      localeCode:
+        type: string
+        description: Measurement family label for the locale `localeCode`
+  standard_unit_code:
+    type: string
+    description: Unit code used as the standard unit for this measurement family
+  units:
+    type: array
+    description: Family units
+    items:
+      type: object
+      properties:
+        code:
+          type: string
+          description: Measure code
+          x-immutable: true
+        labels:
+          type: object
+          description: Unit labels for each locale
+          x-validation-rules: The `localeCode` is the code of an existing and activated locale
+          default: {}
+          properties:
+            localeCode:
+              type: string
+              description: Unit label for the locale `localeCode`
+        convert_from_standard:
+          type: array
+          description: Mathematic operation to convert the unit into the standard unit
+        symbol:
+          type: string
+          description: Measure symbol
+example: {
+   "code": "AREA",
+   "labels": {
+     "en_US": "Area",
+     "fr_FR": "Surface"
+   },
+   "standard_unit_code": "SQUARE_METER",
```

Comment:
Should this code be one of the units defined below?

Reply:
yes

Reply:
@Delphineray you need to add this unit in the units array below.

Reply:
done